specification/synapse/resource-manager/Microsoft.Synapse/preview/2020-04-01-preview/sqlDatabase.json

```diff
@@ -25,19 +25,19 @@
         "operationId": "SqlDatabases_Get",
         "parameters": [
           {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
           },
           {
-            "$ref": "../../../../common/v1/types.json#/parameters/WorkspaceNameParameter"
+            "$ref": "#/parameters/WorkspaceNameParameter"
           },
           {
-            "$ref": "../../../../common/v1/types.json#/parameters/SqlDatabaseNameParameter"
+            "$ref": "#/parameters/SqlDatabaseNameParameter"
           },
           {
-            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
```

Comment:
Why do you change this reference?  reference to common-types is best practise

Reply:
Done